src/Command/Api/ApiCommandBase.php

```diff
@@ -41,7 +41,7 @@ class ApiCommandBase extends CommandBase {
    */
   protected function execute(InputInterface $input, OutputInterface $output) {
     // Build query from non-null options.
-    $acquia_cloud_client = $this->getApplication()->getContainer()->get('cloud_api')->getClient();
+    $acquia_cloud_client = $this->clientService->getClient();
```

Comment:
I'd like to name this something like cloudApiService, since you could easily have clients for other APIs like Pipelines API.